_posts/2025-09-08-data-and-florence-trends.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Come e dove crescono gli Airbnb a Firenze?"
+date: 2025-09-18
+author: "Bernardo & Martina"
+tags: [italia, florence, time-trend]
+categories: [italiano]
```

Comment:
Che ne dici di usare solo questa come categoria? Possiamo usarne un altra per l'inglese così i post tradotti li mettiamo sotto quella, che secondo me mettere due lingue in un post è scomodo. Le categorie si possono far diventare bottoni nella navigation in alto quindi avremmo due sezioni insomma